### PR TITLE
Fixes #540 Add CNAME in gh-pages deployment

### LINKED ISSUE #540: Add CNAME in gh-pages deployment
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Expected behavior:**
Badgeyay Front end will be accessible at that CNAME i.e `badgeyay.com`

Working on it

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #540 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Add CNAME in gh-pages deployment
-

### CODE REVIEW

File: frontend/config/deploy.js [hunk 1]
```diff
@@ -6,13 +6,20 @@ module.exports = function(deployTarget) {
...
-     ENV.git = {
-       repo: `https://ParthS007:${process.env.GH_TOKEN}@github.com/fossasia/badgeyay`,
-       branch: 'gh-pages',
-       commitMessage: 'Deployed %@'
+if (deployTarget === 'gh-pages-with-domain' || deployTarget === 'gh-pages') {
+  ENV.pipeline = {
+    disabled: {}
+  };
+  ENV.git = {
+    repo: `https://ParthS007:${process.env.GH_TOKEN}@github.com/fossasia/badgeyay`,
```
Comment (vaibhavsingh97) [inline frontend/config/deploy.js]: I think, it's should be `fossasia` instead of `Parth`

File: frontend/config/deploy.js [hunk 2]
```diff
@@ -6,15 +6,22 @@ module.exports = function(deployTarget) {
...
-       branch: 'gh-pages',
-       commitMessage: 'Deployed %@'
-     };
-   }
+  if (deployTarget === 'gh-pages-with-domain' || deployTarget === 'gh-pages') {
+    ENV.pipeline = {
+      disabled: {}
+    };
+    ENV.git = {
+      repo: `https://ParthS007:${process.env.GH_TOKEN}@github.com/fossasia/badgeyay`,
```
Comment (S2606) [inline frontend/config/deploy.js]: I think change parthS007 to fossasia

Comment (S2606) [reply frontend/config/deploy.js]: Ohh sorry @vaibhavsingh97 already mentioned it before

**Comment by ParthS007** [comment]
Thanks @S2606  👍

**Comment by ParthS007** [comment]
@vaibhavsingh97  Please see gh-pages deployment of `susper` , `loklak_search` , `open-event-front-end` and other repo where gh-pages auto deployment is active 👍

**Comment by ParthS007** [comment]
Merging it 👍

**Pull request merged**